### DISCUSSION TITLE
Allow hiding/disabling of the service graph tab in tempo

### DISCUSSION THREAD

**Discussion by robertlaurin** [grafana/grafana#56907] [General]
It would be nice if we could hide the Tempo service graph when it is not configured. [A similar configuration exists for the trace search tab that allows hiding it.](https://grafana.com/docs/grafana/latest/datasources/tempo/#search)

<img width="512" alt="image" src="https://user-images.githubusercontent.com/23463253/195709187-c8fed7e4-36b7-44fa-9298-c694c02ce088.png">

Related: https://github.com/grafana/grafana/pull/52375

It is possible this configuration already exists and I was unable to find it, if so feel free to point me in the right direction and close the issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
